modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -3,7 +3,7 @@
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-creating-bridge-nad-cli_{context}"]
+[id="virt-creating-linux-bridge-nad-cli_{context}"]
 = Creating a Linux bridge network attachment definition in the CLI
 
 As a network administrator, you can configure a network attachment definition of type `cnv-bridge` to provide Layer-2 networking to pods and virtual machines.
```

Comment:
Small nit: "layer-2" or "layer 2" is generic, so it should not be capitalized. I suspect that it is capitalized elsewhere in our docs but it looks weird to me. I have only seen the "L" capitalized when layer 2 is shortened to "L2".

Reply:
Done. The web indicates "layer-2" is standard.